server/container/guice/guice-common/src/main/java/org/apache/james/utils/RetryExecutorUtil.java

```diff
@@ -23,15 +23,16 @@
 
 public class RetryExecutorUtil {
 
-    public static final int INITIAL_DELAY_MILLIS = 500;
-    public static final int MULTIPLIER = 2;
-
+    private static final int INITIAL_DELAY_MILLIS = 5000;
+    private static final int MULTIPLIER = 2;
+    private static final int MAX_DELAY_MILLIS = 5000 * 1024;
```

Comment:
Reuse INITIAL_DELAY_MILLIS

5s between each retries seems a lot, I prefer 500.

Why 1024? If failing 5s * 1024 is more than an hour, which is rather long!

I also believe we don't need to specify maxDelay, as we have maxRetry + minDelay...